detected_object_cartesian[] objects